pkg/cmos/cmos.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmos
+
+import (
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+func Read(reg uint64, data memio.UintN) error {
+        regVal := memio.Uint8(reg)
+        if err := memio.Out(0x70, &regVal); err != nil {
+                return err
+        }
+        return memio.In(0x71, data)
+}
+
+func Write(reg uint64, data memio.UintN) error {
+        regVal := memio.Uint8(reg)
+        if err := memio.Out(0x70, &regVal); err != nil {
+                return err
+        }
+        return memio.Out(0x71, data)
+}
+
```

Comment:
Just curious, how did you test these? Not blocking for this PR, but I'm wondering if we could add VM tests or something at some point.

Reply:
I'll go a little stronger, to be honest - if there is cmos in qemu, please do write a VM test before merging.

Reply:
Manually tested on qemu and on my machine. I'll work on some vm tests for it.